compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/codegen/IrSourceCompilerForInline.kt

```diff
@@ -104,7 +105,10 @@ class IrSourceCompilerForInline(
         return FunctionCodegen(lambdaInfo.function, codegen.classCodegen).generate(codegen, reifiedTypeParameters)
     }
 
-    override fun compileInlineFunction(jvmSignature: JvmMethodSignature, callDefault: Boolean, asmMethod: Method): SMAPAndMethodNode =
+    override fun inlineFunctionSignature(jvmSignature: JvmMethodSignature, callDefault: Boolean): Pair<ClassId, Method>? =
+        callee.parentClassId?.let { it to jvmSignature.asmMethod }
```

Comment:
this is broken actually
1. forgot to replicate compatibility hack for old inline class ABI;
2. for some reason when `callee` is an accessor of a top-level property in a multifile class, `parentClassId` is wrong (points to the facade instead of the real implementation class; for functions it's correct though)

Reply:
Never mind 1, that awful hack is covered by [at the IR lowering level](https://github.com/JetBrains/kotlin/blob/master/compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/inlineclasses/MemoizedInlineClassReplacements.kt#L233).

Reply:
Fixed 2 at the source.